packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -156,6 +156,7 @@ contract OptimisticOracle is OptimisticOracleInterface, Testable, Lockable {
         require(getState(msg.sender, identifier, timestamp, ancillaryData) == State.Invalid, "requestPrice: Invalid");
         require(_getIdentifierWhitelist().isIdentifierSupported(identifier), "Unsupported identifier");
         require(_getCollateralWhitelist().isOnWhitelist(address(currency)), "Unsupported currency");
+        require(timestamp <= getCurrentTime().add(90), "Timestamp in future"); // Give 90 seconds of leeway.
```

Comment:
I think this deserves a comment or some more explanation why this just can't be `timestamp <= getCurrentTime()`. Just explaining the situation where `timestamp > currentTime`

Reply:
As discussed offline, we can just get rid of the 90 second window.